hicommonwealth/commonwealth
# Migrated `/viewCount` calls to react query

## Link to Issue
Closes: https://github.com/hicommonwealth/commonwealth/issues/8250

## Description of Changes
Migrated `/viewCount` calls to react query with a 5sec cache value.

## "How We Fixed It"
N/A

## Test Plan
- View any thread page
- Verify you see the view count in the thread metadata section

## Deployment Plan
N/A

## Other Considerations
N/A
